django/utils/autoreload.py

```diff
@@ -370,6 +370,19 @@ def stop(self):
 class StatReloader(BaseReloader):
```

Comment:
Should `WatchmanReloader` be updated too? If that has to be updated too, I think most of the code added here can be moved to `BaseReloader`.